.github/workflows/merge.yml

```diff
@@ -0,0 +1,86 @@
+name: Merge to develop
+
+on:
+  push:
+    branches: [develop]
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  actions: read
+
+concurrency:
+  group: merge-develop
+  cancel-in-progress: false
+
+jobs:
+  bump_version:
+    name: Bump version (skip if already bumped)
+    runs-on: ubuntu-latest
+    if: ${{ github.actor != 'AVSlabBot' }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: develop
+          fetch-depth: 0
+          token: ${{ secrets.BOT_ACCESS_TOKEN }}
+
+      - name: Bump version
+        run: ./.github/workflows/version-bumper.sh ./docs/source/bskVersion.txt
+
+      - name: Commit and push
+        run: |
+          git config user.name "AVSlabBot"
+          git config user.email "cuavslab@gmail.com"
+          git commit -a -m "[AUTO] Bump version number" || echo "No changes"
+          git push || true
+
+  build-ubuntu-latest-wheels:
```

Comment:
Once the wheels are running, we can then remove this custom wheel generation.

Reply:
Correct, my other PR which is based on this one removes this section entirely.